src/core/config_mod.F90

```diff
@@ -223,7 +223,7 @@ SUBROUTINE Config_Chem_State( filename, GridState, ChemState, RC )
       integer :: n
       CHARACTER(LEN=QFYAML_NamLen) :: key
       CHARACTER(LEN=QFYAML_StrLen) :: v_str
-      integer :: v_int
+      ! integer :: v_int
```

Comment:
why not delete these instead of commenting them out? if they are needed we could add them back in